internal/Network/DNS/Types/Internal.hs

```diff
@@ -998,6 +998,8 @@ data RData = RD_A IPv4           -- ^ IPv4 address
                                  -- ^ Child DS (RFC7344)
            | RD_CDNSKEY Word16 Word8 Word8 ByteString
                                  -- ^ Child DNSKEY (RFC7344)
+           | RD_CAA Word8 ByteString ByteString
```

Comment:
Show the `tag` field be a `case-insenstive` ByteString (from `Data.CaseInsensitive`)? Or is case-insensitive matching best left to users?